stories/setup/config.js

```diff
@@ -1,11 +1,42 @@
-import { configure } from '@storybook/react';
+import { configure, setAddon } from '@storybook/react';
 import { setOptions } from '@storybook/addon-options';
+import chaptersAddon, { setDefaults } from 'react-storybook-addon-chapters';
+import infoAddon, {
+  setDefaults as setAddonInfoDefaults,
+} from '@storybook/addon-info';
+
+// TBD: overrides chapters defaults. Do we want this?
+setDefaults({
+  sectionOptions: {
+    allowPropTablesToggling: false,
+    allowSourceToggling: true,
+    showSource: false,
+    useTheme: false,
```

Comment:
I sort of went into why I left these this way here: https://github.com/mozilla/addons-frontend/pull/6256#discussion_r218960154

actually I turned source toggle on but I personally think we don't need this here. 

Regarding the display of props, it doesn't seem that this works with flow type as far as I can tell so I set up addon-info for this (on it's own page).